build/teamcity/cockroach/nightlies/sqllogic_corpus_nightly_impl.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
+source "$dir/teamcity-bazel-support.sh"  # For process_test_json
+source "$dir/teamcity-support.sh"  # For process_test_json
+
+bazel build //pkg/cmd/bazci //pkg/cmd/github-post //pkg/cmd/testfilter --config=ci
+BAZEL_BIN=$(bazel info bazel-bin --config=ci)
+google_credentials="$GOOGLE_EPHEMERAL_CREDENTIALS"
+
+log_into_gcloud
+
+ARTIFACTS_DIR=/artifacts
+GO_TEST_JSON_OUTPUT_FILE=$ARTIFACTS_DIR/test.json.txt
+GO_TEST_GEN_JSON_OUTPUT_FILE=$ARTIFACTS_DIR/test-gen.json.txt
+GO_TEST_GEN_CCL_JSON_OUTPUT_FILE=$ARTIFACTS_DIR/test-gen-ccl.json.txt
+GO_TEST_VALIDATE_JSON_OUTPUT_FILE=$ARTIFACTS_DIR/test-validate.json.txt
+GO_TEST_JSON_OUTPUT_FILE_MIXED=$ARTIFACTS_DIR/test-mixed.json.txt
+GO_TEST_VALIDATE_JSON_OUTPUT_FILE_MIXED=$ARTIFACTS_DIR/test-validate-mixed.json.txt
+
+mkdir -p $ARTIFACTS_DIR/corpus
+mkdir -p $ARTIFACTS_DIR/corpus-mixed
+exit_status=0
+
+# Generate a corpus for all non-mixed version variants
+for config in local multiregion-9node-3region-3azs; do
+$BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci --config=ci \
```

Comment:
`bazci` is going to get a breaking change to its CLI, so you have to tweak these calls. Please move the `--` from after the target to after `$BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci`, so it'll be like:

```
BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci -- --config=ci
    test //pkg/sql/logictest/tests/$config/... \
    --test_arg=--declarative-corpus=$ARTIFACTS_DIR/corpus \
    --test_env=GO_TEST_WRAP_TESTV=1 \
    --test_env=GO_TEST_WRAP=1 \
    --test_env=GO_TEST_JSON_OUTPUT_FILE=$GO_TEST_JSON_OUTPUT_FILE.$config \
    --test_timeout=7200 \
    || exit_status=$?
```